analysis_options.yaml

```diff
@@ -4,4 +4,4 @@ analyzer:
     strict-inference: true
     strict-raw-types: true
 
-include: package:flame_lint/analysis_options.yaml
+include: package:flame_lint/analysis_options.yaml
```

Comment:
Super nit, but it should have a new line in the end

Reply:
oh sorry I was playing around with it and accidentally removed. not intentional